src/components/Login.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import auth from "../modules/auth";
+
+const Login = props => {
+  const [displayLoginButton, setDisplayLoginButton] = useState(true);
+  const [loginMessage, setLoginMessage] = useState("");
+
+  const onLogin = event => {
+    event.preventDefault();
+    auth
+      .signIn(event.target.email.value, event.target.password.value)
+      .then(userDatas => {
+        props.changeAuth(true);
+        setLoginMessage(`Logged in as: ${userDatas.data.email}`);
+      })
+      .catch(error => {
+        alert("Invalid login credentials");
+        window.location.reload();
+      });
+  };
+
+  const onLogout = () => {
+    auth
+      .signOut()
+      .then(() => {
+        props.changeAuth(false);
+        setDisplayLoginButton(true);
+      })
+      .catch(error => {
+        setLoginMessage(error);
+      });
+  };
+
+  let loginButton;
+
+  switch (true) {
+    case displayLoginButton && !props.authenticated:
+      loginButton = (
+        <button id="loginButton" onClick={() => setDisplayLoginButton(false)}>
+          Login
+        </button>
+      );
+      break;
+    case !displayLoginButton && !props.authenticated:
+      loginButton = (
```

Comment:
It's a bit weird to call this `loginButton` as it's only a login button in the first out of the three cases

Reply:
changed name to loginFunction